modules/objdetect/misc/python/test/test_charuco_board.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import print_function
+
+import os, tempfile, numpy as np
+
+import sys
+sys.path.append("../../../../../doc/pattern_tools")
```

Comment:
It' fragile solution. `NewOpenCVTests` has `repoPath` filed that defiles main repo root. I propose to use it and skip the related test with warning, if `repoPath` is not defined. See https://github.com/opencv/opencv/blob/4.x/modules/python/test/tests_common.py#L25